southworks/botframework-solutions
# [TypeScript][VA/Skill] Integrate specific cultures

## Purpose

_What is the context of this pull request? Why is it being done?_

We updated the _Virtual Assistant_ and the _Skill_ language cultures to improve the functionality of the **multilanguage** implementation. 

## Changes
_Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)_

- Rename different folder and files with a specific _locale_ name
- Use of the specific _locale_ name for the _deployment scripts_ files
- Changes in _mainDialog_ implementation

## Tests
_Is this covered by existing tests or new ones? If no, why not?_
*-*

## Testing steps

1. Deploy a Virtual Assistant with both languages (eg. _en-us, es-mx_).
2. Verify in the bot solution that the resource folder contains the specific cultures.
3. Execute the Virtual Assistant with any of the deployed specific cultures
4. Check the behavior of the Virtual Assistant communication

## Feature Plan
_Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues._
*-*

### Checklist

#### General

#### Bots

#### Deployment Scripts
